src/sentry/static/sentry/app/components/organizations/timeRangeSelector/index.jsx

```diff
@@ -374,13 +380,12 @@ class TimeRangeSelector extends React.PureComponent {
                       onChange={this.handleSelectDateRange}
                       onChangeUtc={this.handleUseUtc}
                     />
-                    {this.state.hasChanges && (
-                      <SubmitRow>
-                        <MultipleSelectorSubmitRow
-                          onSubmit={() => this.handleCloseMenu()}
-                        />
-                      </SubmitRow>
-                    )}
+                    <SubmitRow>
+                      <MultipleSelectorSubmitRow
+                        onSubmit={() => this.handleCloseMenu()}
```

Comment:
We should make sure `handleCloseMenu` is bound but this can be

```suggestion
                        onSubmit={this.handleCloseMenu}
```